### ISSUE TITLE
Add possibility to add/edit spool length

### ISSUE THREAD

**Issue by Hesi-Re** [Donkie/Spoolman#42]
Thanks for your great work with Spoolman!

It would be nice to add spool length and not only spool weight.

**Comment by Donkie** [comment]
You mean you want to be able to specify a used spool length (in mm) instead of used spool weight when creating a new spool?

**Comment by Hesi-Re** [comment]
Yes. "Used spool length" and "available spool length". For example, when creating new spool it's helpful to specify de total spool length (ex: 1000g -> 335,28m), so when printing it subtracts the used filament length, not only the weight.

**Comment by Hesi-Re** [comment]
If I cut 2 m of the filament, it would be nice to subtract it to the spool

**Comment by Donkie** [comment]
That feature is already requested in #30

**Comment by Bahoue** [comment]
Hello and thank you for Spoolman!
The length option is great, would it be possible to choose mm, cm or metre?
